db/seeds.rb

```diff
@@ -5,3 +5,41 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+# Create Seasons
+Season.create(start_date: '2016-02-12 20:58:01 -0500')
```

Comment:
I think a `Time` object would be better than a string here:

```ruby
Time.new(2016, 2, 17, 20, 58, 01)
```

That should even take your timezone into account. Also, consider if you need precision down to the second like that. If not, just get it down to the day.